Valadzhov/date picker new label 8086 (#2213)
IgniteUI/igniteui-docfx: en/components/date_picker.md

<igx-date-picker cancelButtonLabel="cancel" todayButtonLabel="today" [(ngModel)]="date"></igx-date-picker>
```

#### Custom label

> By placing an `igxlabel` inside a `<label>` tag nested in the component, the default label can be changed to a custom one:

Reviewer: ```suggestion
Using the `igxLabel` directive, the default label can be changed to a custom one:
```

Author: Changed to the suggestion.

````html
<!-- datepicker-label.component.html -->
<igx-date-picker>
    <label igxLabel>Custom label</label>
</igx-date-picker>
````

You can see what the custom label will look like in the multi view mode sample found in next section.

> [!NOTE]

> > Currently the `label` and `labelVisibility` properties are in the process of being deprecated. During this period if `labelVisibility` is set to `false` and a custom label is not provided then a default label `Date` will be set.

Reviewer: ```suggestion
> Currently the `label` and `labelVisibility` properties are deprecated. Until they are removed, if `labelVisibility` is set to `false` and a custom label is not provided then a default `Date` label will be set.
```
